cirq/ops/three_qubit_gates.py

```diff
@@ -14,10 +14,12 @@
 
 """Common quantum gates that target three qubits."""
 
-from typing import Optional, Tuple
+from typing import Any, List, Optional, Tuple, Union
 
 import numpy as np
+import sympy
 
+import cirq
```

Comment:
I keep getting a lint warning because of the ParamResolver type annotation. Is it ok to use # pylint: disable=unused-import?

Reply:
Use the `if TYPE_CHECKING` thing used elsewhere in the codebase

Reply:
Ok, added.